src/Sql/Sql/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Upcoming Release
 * Added an optional parameter `HAReplicaCount` to `Restore-AzSqlDatabase`
+* Updated the max value for 'HAReplicaCount' parameter to max int value
```

Comment:
The parameter was just introduced and not released yet, so I don't think we need to log that in the change log. This PR can go without it.
```suggestion
```